rules/type-declaration/tests/Rector/FunctionLike/ReturnTypeDeclarationRector/Fixture/respect_vendor_return_type.php.inc

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Rector\TypeDeclaration\Tests\Rector\FunctionLike\ReturnTypeDeclarationRector\Fixture;
+
+class RespectVendorReturnType
+{
+    /**
+     * @return bool
+     */
+    public function run()
+    {
+        return false;
+    }
+}
+
+class RespectVendorReturnType2 extends RespectVendorReturnType {
+    public function run() {
+       return true;       
+    }
+}
```

Comment:
Fixture should contains only 1 class if possible.
The parent class should be located in `/Source` directory.